### PR TITLE
dmesg: add Chinese translation

### PR DESCRIPTION
**Pull request by xiaolong-666**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by marchersimon** [comment]
@xiaolong-666 do you need help with the CLAsistent?

**Comment by xiaolong-666** [comment]
> @xiaolong-666 do you need help with the CLAsistent?

Sure, i  have agreed to the CLA for tldr-pages/tldr, but the status is still pending

**Comment by marchersimon** [comment]
> longqiang seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account.

Seems like the email you used for your commit isn't added to you GitHub account. You could either add it or change the email of your commit.

**Comment by xiaolong-666** [comment]
> > longqiang seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account.
> 
> Seems like the email you used for your commit isn't added to you GitHub account. You could either add it or change the email of your commit.

thanks, i did not add it to my account

**Comment by CleanMachine1** [comment]
You can tick the 
>  The page has 8 or fewer examples.
since it is 7

**Comment by xiaolong-666** [comment]
> since it is 7
I think one of the examples is problematic，so removed it. If you need to be consistent, I can add it back

**Comment by CleanMachine1** [comment]
Even so, 8 is still acceptable

File: pages.zh/linux/dmesg.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# dmesg
+
+> 显示或控制内核环形缓冲区。
+> 更多信息：<https://man7.org/linux/man-pages/man1/dmesg.1.html>。
```
Comment (bl-ue) [inline pages.zh/linux/dmesg.md:4]: We're working on transitioning from man7.org to manned.org (#5790)
```suggestion
> 更多信息：<https://manned.org/dmesg>。
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by navarroaxel — COMMENTED** [review]
You forgot translate this example:
```
dmesg | grep -i memory
```

**Comment by xiaolong-666** [comment]
> You forgot translate this example:
> 
> ```
> dmesg | grep -i memory
> ```

Well, i should just do translation

File: pages.zh/linux/dmesg.md (hunk 1 above)
Comment (xiaolong-666) [reply pages.zh/linux/dmesg.md:4]: Okay, I will use this link in the future

**Review by marchersimon — APPROVED** [review]
Looks good, thanks @xiaolong-666

**Review by navarroaxel — APPROVED** [review]

File: pages.zh/linux/dmesg.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# dmesg
+
+> 显示或控制内核环形缓冲区。
+> 更多信息：<https://manned.org/dmesg>。
```
Comment (bl-ue) [inline pages.zh/linux/dmesg.md:4]:
```suggestion
> 更多信息：<https://manned.org/dmesg>.
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by bl-ue — APPROVED** [review]

**Review by CleanMachine1 — APPROVED** [review]

**Comment by bl-ue** [comment]
Oh I didn't even notice that you approved @CleanMachine1.  I would've let you merge :)

**Comment by CleanMachine1** [comment]
I clicked approve as you merged

**Pull request merged**